### ISSUE TITLE
is it a python2 project?

### ISSUE THREAD

**Issue by rick-wang-musetalk** [f90/AdversarialAudioSeparation#6]
is it a python2 project? i have errors such as import cPickle and audiolab.

**Comment by f90** [comment]
Yes this is Python2 I am fairly sure. If you give me more specifics about the errors I can fix them to make it Python 3 compatible!
